src/snowflake/cli/plugins/spcs/compute_pool/commands.py

```diff
@@ -18,11 +18,16 @@
 )
 
 
+ComputePoolNameArgument = typer.Argument(
+    ..., help="Name of the compute pool.", callback=object_name_callback
+)
```

Comment:
actually compute pool can only be a single valid identifier since it's account level object